services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -3227,6 +3227,14 @@ public void setStoreCurrentVersion(String clusterName, String storeName, int ver
     });
   }
 
+  /**
+   * Unsupported operation in the child controller.
+   */
+  @Override
+  public void rollbackToBackupVersion(String clusterName, String storeName) {
```

Comment:
Shall we support it in case customer wants us to do a rollback for a partial succeed push job?

Reply:
Version rollback is already supported in child controller: setStoreCurrentVersion. Operators could use `--set-version` command with child controller url, cluster, store, and version number to rollback. I updated the PR to implement rollbackToBackupVersion in child controller in case in the future we want to support child controller version rollback without specifying version number.